vms/administrator/urls.py

```diff
@@ -1,15 +1,11 @@
 # Django
-from django.conf.urls import patterns, url
+from django.conf.urls import url
 
 # local Django
 from administrator import views
 from administrator.views import GenerateReportView, AdminUpdateView, ProfileView
 
-urlpatterns = patterns(
-    '',
+urlpatterns = [
     url(r'^report/$', GenerateReportView.as_view(), name='report'),
-    url(r'^settings/$', views.settings, name='settings'),
-    url(r'^edit/(?P<admin_id>\d+)$', AdminUpdateView.as_view(), name='edit'),
```

Comment:
Did you forget to add this url?

Reply:
I guess I haven't pulled all the code. Let me do the same.